### ISSUE TITLE
NPE when trying to send message

### ISSUE THREAD

**Issue by snimavat** [orkonano/grails-mailgun#11]
Just installed the 2.0.0 version and when trying to send a message, i get following error

```
Caused by: java.lang.NullPointerException: Cannot invoke method render() on null object
    at grails.plugin.mailgun.MailgunService.sendMessage(MailgunService.groovy:39) ~[grails-mailgun-2.0.0.jar:na]
```

Looks like you forgot to register the DefaultEmailHtmlRender as a spring bean ?

RestBuilder is also null

And after manually resolving all this errors by registering required beans in my Application.groovy
I am finally stuck here

``````

Caused by: java.lang.ClassCastException: org.springframework.util.LinkedMultiValueMap cannot be cast to groovy.util.slurpersupport.GPathResult
    at grails.plugins.rest.client.RestBuilder.invokeRestTemplate(RestBuilder.groovy:310) ~[grails-datastore-rest-client-5.0.0.RC2.jar:na]
    at grails.plugins.rest.client.RestBuilder.doRequestInternal(RestBuilder.groovy:274) ~[grails-datastore-rest-client-5.0.0.RC2.jar:na]
    at grails.plugins.rest.client.RestBuilder.doRequestInternal(RestBuilder.groovy:262) ~[grails-datastore-rest-client-5.0.0.RC2.jar:na]
    at grails.plugins.rest.client.RestBuilder.post(RestBuilder.groovy:148) ~[grails-datastore-rest-client-5.0.0.RC2.jar:na]
    at grails.plugin.mailgun.MailgunService.sendMessage(MailgunService.groovy:49) ~[grails-mailgun-2.0.0.jar:na]```
``````

**Comment by orkonano** [comment]
@snimavat 

> Caused by: java.lang.ClassCastException: org.springframework.util.LinkedMultiValueMap cannot be cast to groovy.util.slurpersupport.GPathResult

This error is relative to #10 . I resolved in migrate-grails3.1 branch (https://github.com/orkonano/grails-mailgun/tree/migrate-grails3.1)
Today I'll try to push into master branch

About the NPE apparently the migration to grails 3.x broke this part.
In my private app I register into resources.groovy 

```
restBuilder(RestBuilder, [:])

 emailHtmlRender(BigHamletEmailRender){
      emailRenderService = ref('emailRenderService')
 }
```

**Comment by snimavat** [comment]
@orkonano Thanks for fix - is java.lang.ClassCastException fixed too ?

**Comment by orkonano** [comment]
@snimavat yes! is fix int #10 

You have to update the dependency to 2.0.2

http://grails-plugins.org/#plugin/grails-mailgun
